src/protocol/http/src/test/java/org/apache/jmeter/protocol/http/sampler/TestHTTPSamplers.java

```diff
@@ -371,6 +372,7 @@ public void testSetAndGetMimetype() {
     }
 
     @Test
+    @Ignore("getFileServer().setBase is not thread-safe, so the test should be refactored somehow")
```

Comment:
We could `implement JMeterSerialTest` instead of just ignoring this test but keep this as a comment?
At some point all things implementing JMeterSerialTest should be fixed.